src/Cake.Common/Tools/DotNet/Store/DotNetStoreSettings.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.DotNet.Store
+{
+    /// <summary>
+    /// Contains settings used by <see cref="DotNetStorer" />.
+    /// </summary>
+    public class DotNetStoreSettings : DotNetSettings
+    {
+        /// <summary>
+        /// Gets or sets the output directory.
+        /// </summary>
+        public DirectoryPath OutputDirectory { get; set; }
+
+        /// <summary>
+        /// Gets or sets the version of the installed Shared Framework to use to run the application.
+        /// </summary>
+        public string FrameworkVersion { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to skip the optimization phase.
+        /// </summary>
+        public bool SkipOptimization { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to skip symbol generation.
+        /// </summary>
+        /// <remarks>
+        /// Currently, you can only generate symbols on Windows and Linux.
+        /// </remarks>
+        public bool SkipSymbols { get; set; }
```

Comment:
Args in latest SDK missing
* `--use-current-runtime                    Use current runtime as the target runtime.`
* `--working-dir <WORKING_DIR>          The working directory used by the command to execute.`

Reply:
There is `WorkingDirectory` from `ToolSettings`

Reply:
`WorkingDirectory` agree just noticed there was an option for it unsure if there's any semantical difference or not.

`--use-current-runtime` seems to be a common option

https://github.com/dotnet/sdk/blob/df5c30db37d58380f5705250d76552979e4f3375/src/Cli/dotnet/CommonOptions.cs#L59